docs/advanced-configuration.md

```diff
@@ -539,6 +539,10 @@ Note: Running the collector agent for log collection in non-root mode is not cur
 [related GitHub feature request issue](https://github.com/signalfx/splunk-otel-collector-chart/issues/891).
 
 ## Network explorer
+
+**NOTE:** Network Explorer related features should be considered to have an ALPHA maturity level and be experimental.
```

Comment:
Do we document maturity levels [like the collector ](https://github.com/open-telemetry/opentelemetry-collector#alpha) does? If yes, can we link to it with `ALPHA maturity` here?

Reply:
I like that. Done. We do follow the otel maturity levels (as far as I know). Will need to link other chart component docs in the future for uniformity.